device/mellanox/x86_64-nvidia_sn4280-r0/ACS-SN4280/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,44 @@
+{#
+    Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+    Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '300m' %}
+{%-set ports2cable = {
+            'spinerouter_leafrouter' : '2000m',
+            'leafrouter_torrouter' : '300m'
+       }
+-%}
+{% set ingress_lossless_pool_size =  '46743552' %}
+{% set ingress_lossless_pool_xoff  =  '8781696' %}
+{% set egress_lossless_pool_size =  '60817392' %}
+{% set egress_lossy_pool_size =  '46743552' %}
```

Comment:
Why different pool size and xoff values for T0 and T1? Any additional reserved buffers other than the number of ports and cable-length?

Reply:
T1-T2: 2000m
T0-T1: 300m
T0-server: 40m

Cable lengths for uplink and downlink is different for T1 and T0 and so the variance in the values.